Node/core/src/cards/OAuthCard.ts

```diff
@@ -71,4 +73,41 @@ export class OAuthCard implements IIsAttachment {
     public toAttachment(): IAttachment {
         return this.data;
     }
+
+    public static create(connector: ChatConnector, session: Session, connectionName: string, text: string, buttonTitle: string, done: (err: Error, message: Message) => void): void {
+        var msg = new Message(session);
+
+        var asSignInCard: boolean = false;
+        switch (session.message.address.channelId) {
+            case 'msteams':
+            case 'cortana':
+            case 'skype':
+            case 'slypeforbusiness':
```

Comment:
Typo. :-)

Reply:
Yes! SlypeForBusiness is not a new product...